espnet/nets/pytorch_backend/transducer/transformer_decoder.py

```diff
@@ -213,8 +205,8 @@ def recognize(self, h, recog_args):
 
         return [hyp]
 
-    def recognize_beam(self, h, recog_args, rnnlm=None):
-        """Beam search implementation for transformer-transducer.
+    def recognize_beam_default(self, h, recog_args, rnnlm=None):
+        """Beam search implementation.
```

Comment:
Sorry if it's a dump question. 
Is the beam search method of transformer different from that of rnn? Will you make/unify this beam search like ScorerInterface and BeamSearch class? Basically, we should define it as an external function/class rather than a method of each class to reduce the duplication part. 

(Even if you will, you don't need to do it in this PR. This PR is already too large.)

Reply:
> Is the beam search method of transformer different from that of rnn?

There are some minor differences but the methods is the same.

> Will you make/unify this beam search like ScorerInterface and BeamSearch class?

I guess so, I made an interface at the beginning but I wasn't too fond of it. I'll work on a new one after last beam search is implemented and optimization is done!

Reply:
okay, thanks.